tests/dummy/app/pods/components/user-menu/template.hbs

```diff
@@ -0,0 +1,46 @@
+{{frost-button
+  class=(concat css '-user-button')
+  hook=(concat hookPrefix '-userButton')
+  hookQualifiers=hookQualifiers
+  icon='user'
+  pack='frost-nav'
+  text=username
+}}
+
+{{#frost-popover
+  class='frost-navigation-user-menu'
```

Comment:
Yeah, I think so, because those aren't classes provided by this addon. Of course, maybe I should make the user menu an actual component that this guy provides? Doesn't mean people have to use it, so it won't break anyone, but might make life easier for some.